src/js_native_api_v8.cc

```diff
@@ -3257,3 +3259,12 @@ napi_status NAPI_CDECL napi_is_detached_arraybuffer(napi_env env,
 
   return napi_clear_last_error(env);
 }
+
+napi_status NAPI_CDECL node_api_is_feature_enabled(napi_env env,
```

Comment:
I wonder if this should be in node_api.cc instead of this file. It is not tied to v8 right?

Reply:
It is not tied to V8. I was thinking that we want to have features available for different JS engines.